### PR TITLE
Display collections without descriptions

### PR DESCRIPTION
**Pull request by no-reply**
`CollectionPresenter#description` is delegated to `#solr_document`, which can return `nil`. When we display the collection, we try to call `#each`, which gives `NoMethodError` in the `nil` case.

Using the safe access operator `&.` avoids this bug and allows display of Collections and AdminSets without any descriptions.

Connected to #2575.

Guidance for testing, such as acceptance criteria or new user interface behaviors:
* Create a collection with no description;
* Visit the collection show page and ensure there is no error.

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by no-reply** [comment]
@jcoyne:

> Have you considered having the description return [] if it’s not present?

This would be a nice change, but isn't sufficient here for two reasons:

  - In practice, applications provide their own implementations of `SolrDocument` via the generator; these normally return `[Array<Object>, nil]`. To fix this bug for existing applications, we need to be tolerant.
  - Changing from returning `nil` in the empty case to `[]` would change the method signature for either this one accessor, or (probably better?) all field accessors on `SolrDocument`. It seems reasonable to expect that some users do `.nil?` checks which would be broken on the change.

In short, this feels like a larger `SolrDocument` reworking, and we need to continue to support current behavior until at least 3.0.0.

**Comment by no-reply** [comment]
> The second &. makes sense given your description, but why is the first needed?

I've not seen any cases where `presenter` is `nil`, but the implementation proposed here seems to give a reasonable default for this partial in that case; i.e. don't display any descriptions if no presenter is given.

I could readily be talked into removing it (I also prefer to avoid proliferation of conditionals). I have the sneaking suspicion that I'm going to be back here in a month re-adding it in response to some honeybadger ticket, but maybe that will prompt a better rethink(?). If you want to ask for the change, I'll put it in.
